### ISSUE TITLE
Docker Login Request Forbidden

### ISSUE THREAD

**Issue by aplogicdlamb** [docker/hub-feedback#528]
Description of problem:
Attempt to login returns 403 - Request forbidden by administrative rules.

`docker version`: 
<pre><code>Client:
 Version:      1.91
 API version:  1.21
 Go version:   go1.4.3
 Git commit:   a34a1d5
 Built:        Fri Nov 20 17:56:04 UTC 2015
 OS/Arch:      windows/amd64

Server:
 Version:      1.9.1
 API version:  1.21
 Go version:   go1.4.3
 Git commit:   a34a1d5
 Built:        Fri Nov 20 17:56:04 UTC 2015
 OS/Arch:      linux/amd64</code></pre>

`docker info`:
<pre><code>
Containers: 4
Images: 14
Server Version: 1.9.1
Storage Driver: aufs
 Root Dir: /mnt/sda1/var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 22
 Dirperm1 Supported: true
Execution Driver: native-0.2
Logging Driver: json-file
Kernel Version: 4.1.13-boot2docker
Operating System: Boot2Docker 1.9.1 (TCL 6.4.1); master : cef800b - Fri Nov 20 19:33:59 UTC 2015
CPUs: 1
Total Memory: 996.2 MiB
Name: default
ID: 7COW:HAU5:COLA:WOUP:GBDU:TPTA:TH6G:WDKO:DMHZ:FZ3S:TQ4Y:SA66
Debug mode (server): true
 File Descriptors: 11
 Goroutines: 20
 System Time: 2016-01-06T00:13:56.519947108Z
 EventsListeners: 0
 Init SHA1:
 Init Path: /usr/local/bin/docker
 Docker Root Dir: /mnt/sda1/var/lib/docker
Labels:
 provider=virtualbox
</code></pre>

`uname -a`:
`MINGW32_NT-6.2 [MyWorkstationName] 1.0.12(0.46/3/2) 2012-07-05 14:56 i686 unknown`

Environment details (AWS, VirtualBox, physical, etc.):
VirtualBox 5.0.12 r104815
Windows 8.1 Pro

How reproducible:
Every time

Steps to Reproduce:
1. From the prompt, type `docker login --username=[user] --email=[email]`.
2. Enter password when prompted.

Actual Results:
<pre><code>Error response from daemon: Unexpected status code [403] : &lt;html>&lt;body>&lt;h1>403 Forbidden&lt;/h1>
Request forbidden by administrative rules.
&lt;/body>&lt;/html></code></pre>

Expected Results:
Successful login, as per the documentation; (Windows, step 6)
https://docs.docker.com/windows/step_six/

**Comment by aplogicdlamb** [comment]
As per a Docker forum message (https://forums.docker.com/t/docker-login-returns-403-forbidden-by-administrative-rules/4291/5), I typed `docker-machine ssh default`. It changed the command prompt from `[domainUsername]@[machineName]` to `docker@default:`, and now I can login, but not I have this other prompt. Is that normal? Is that what it should be? I activated my session from the Docker Quickstart Terminal.
